### PR TITLE
fix: disable ACL in dev-mode

### PR DESCRIPTION
**Pull request by octonato**
References https://github.com/lightbend/kalix/issues/7783

### CODE REVIEW

File: samples/js/js-doc-snippets/docker-compose.yml [hunk 1]
```diff
@@ -11,6 +11,13 @@ services:
       USER_FUNCTION_HOST: ${USER_FUNCTION_HOST:-host.docker.internal}
       USER_FUNCTION_PORT: ${USER_FUNCTION_PORT:-8080}
       PUBSUB_EMULATOR_HOST: gcloud-pubsub-emulator
+      # Comment to enable ACL check in dev-mode
+      # see https://docs.kalix.io/services/using-acls.html#_local_development_with_acls
+      ACL_ENABLED: 'false'
+      # Uncomment to disable the JWT dev secret
+      # JWT_DEV_SECRET: "false"
+      # Uncomment to set the JWT dev secret issuer
+      # JWT_DEV_SECRET_ISSUER: "my-issuer"
```
Comment (octonato) [inline samples/js/js-doc-snippets/docker-compose.yml:20]: some were missing the JWT settings

**Comment by johanandren** [comment]
Seems the integration tests is testing ACLs so need a change in one way or the other.

**Comment by octonato** [comment]
@johanandren, test is passing now. My bad, I changed something in the integration tests that should have been left untouched.

**Review by johanandren — APPROVED** [review]

**Comment by johanandren** [comment]
This needs an extra note in announcement of release since it might break user tests.

**Comment by octonato** [comment]
yes, it's on my list. I also need to add it to the main docs.

**Pull request merged**